vllm/worker/habana_model_runner.py

```diff
@@ -812,6 +812,9 @@ def warmup_scenario(self, batch_size, seq_len, is_prompt, kv_caches) -> None:
 
     @torch.inference_mode()
     def warmup_model(self, kv_caches: List[torch.Tensor]) -> None:
+        if os.environ.get('VLLM_SKIP_WARMUP', 'false').lower() == 'true':
```

Comment:
Since this is a device specific feature/env what do you think about renaming it to `SKIP_HPU_WARMUP`?

Reply:
I guess we need to figure out a naming scheme, because we also have "VLLM_HPU*" flags... I'd keep it as is and then figure out how we want to handle such env flags and clean it up.